api/app/tests/apps/api/v1/test_private_signal_endpoint_create.py

```diff
@@ -300,3 +300,19 @@ def test_create_initial_child_signals_copy_attachments(self, validate_address):
 
             self.assertEqual(len(detail_response_json['attachments']), 1)
             self.assertJsonSchema(self.retrieve_signal_schema, detail_response_json)
+
+    @patch('signals.apps.api.v1.validation.address.base.BaseAddressValidation.validate_address',
+           side_effect=AddressValidationUnavailableException)  # Skip address validation
+    def test_create_initial_signal_interne_melding(self, validate_address):
+        signal_count = Signal.objects.count()
+
+        initial_data = copy.deepcopy(self.initial_data_base)
+        initial_data['reporter']['email'] = 'test-email-1' \
+                                            f'{settings.API_TRANSFORM_SOURCE_BASED_ON_REPORTER_DOMAIN_EXTENSIONS[0]}'
```

Comment:
Nice that you access the setting, but you should use a settings override --- `API_TRANSFORM_SOURCE_BASED_ON_REPORTER_DOMAIN_EXTENSIONS` may not contain anything.

Reply:
I do not mind if a test will fail because of missing or misconfigured settings

Reply:
not a blocker